src/applicative.rs

```diff
@@ -0,0 +1,37 @@
+use kind::Kind;
+use functor::Functor;
+
+pub trait Applicative<B, F>: Functor<B> + Kind<F> + Sized
+    where F: Fn(Self::Item) -> B {
+    fn pure_(Self::Item) -> Self;
```

Comment:
yuck on the hack for avoiding the keyword.  I might suggest looking to other langauges for naming precedent.  (I know a lot of JS libs call this `of`; I don't know about other languages)

ISTM you are also going to immediately run into problems with type inference when you try to use this method, because its signature does not constrain `F`.

Reply:
Some "insider info:" `pure` will likely be removed as a keyword soon-ish.

For now, I'm partial to `pointed` or `wrap`.